projects/2019_12_14_pkt-whitepaper-website.md

```diff
@@ -0,0 +1,170 @@
+
+* Assistance with Questions: Throughout the process, I am available to discuss and clarify the questions and to brainstorm about the answers. However, I endeavour not to “lead the witness” and tell you what I think. My job, as I see it, is to be a neutral “integrating synthesist” who builds a house out the raw materials of the answers to the 54 questions.
+
+* Within 30 calendar days of receiving the 54 approved answers and any relevant supporting materials, I will deliver a first draft white paper. Of the 16 that I have done, half of them were so in tune with client expectations and requirements that there were zero changes requested of me and the first draft was put the work. The others went through one to three rounds of comments until they were finished. Each round took about two to four days on each side for the revisions. 
+
+* Congratulations. You now have your completed PKT white paper!
+
+* Powerpoint Deck: I will take the details of the whitepaper and compile them into a deck to use as an evangelist of the project on a global scale. 
+
+* Presentation video: Working with Jesse and Josh, I will create a video presentation of the deck to be displayed on the website. This video will help users understand the project and excite them to dig in deeper.
+
+#### Whitepaper Milestones:
+
+Alex Lightman (AL) provides 54 questions to PoC. (1 day)
+PoC gets the “best” people to give their best answers to the questions. (2 weeks)
+PoC arranges and sends approved answers to AL. (1 day)
+1.0 version of the WP is delivered by AL to the PoC. (30 days)
+Revisions are made if and as needed. (2 weeks)
+White paper is completed.
+AL makes a PowerPoint deck based on the new approved WP. (1 week)
+AL creates video recorded presentation of the deck and posts it on his YouTube channel and provides a copy for the PKT community to use to promote the WP. (3 weeks)
+
+**Total time: 81 days**
+
+<br/>
+
+### Website
+
+We wish to build a community-facing website where users can easily access information regarding PKT Cash and adoption. This website will clearly articulate the mission of the project with steps on how to adopt the technology. 
+Custom design: Jesse and Josh will work with their design team to create a custom website design that clearly articulates the brand and messaging of the PKT community.
+
+* Copywriting: Jesse will extract prose from the white paper and adjust the text to fit within the confines of the site. 
+
+* Web development: Josh will work with his development team to take the designs and code them into a functional website.
```

Comment:
Can you provide samples of previous websites this development team has built?

Reply:
For the designer we are working with this company:
https://www.behance.net/dambaagency
https://damba.agency/projects/sad

For development we are working with this company:
https://hipinspire.com